freedom/llh_service/eval_llh.py

```diff
@@ -83,11 +84,25 @@ def apply_boundary_guard(base_llhs, theta, model, bg_lim, invalid_llh):
         boundary guard limit
     invalid_llh: tf.constant
         value returned at points where model(param) <= bg_lim
+    prior: tf.constant
+        should boundary guard also be used as bayesian prior (otherwise just hard limit)
 
     """
-    bg_vals = model(theta)[:, 0]
+    param_limits = np.array([ #sets boundary guard input between 0 and 1
```

Comment:
Is there an easy way to avoid hard-coding these values? Perhaps they could be part of the boundary guard configuration dictionary.

Also, I'd recommend making this a tf.constant rather than a np.array to avoid requiring the additional import in this function.